c2corg_ui/templates/whatsnew.html

```diff
@@ -0,0 +1,42 @@
+<%inherit file="base.html"/>
+<%namespace file="helpers/common.html" import="show_title"/>
+
+<%block name="metarobots">
+  <meta name="robots" content="index,follow">
+</%block>
+
+<%block name="moduleConstantsValues">
+  ## Set in the data request
+  module.value('mapFeatureCollection', null);
+
+</%block>
+
+<section id="whatsnew">
+
+  <header>
+    <h1 class="text-center" translate><span>Feed of the latest changes</span></h1>
```

Comment:
You don't need the span element here. Or at least the translate directive should apply to it, not to the parent, else the span tags are integrated to the string to translate.